CVLByExample/FoundryIntegration/SimpleDappTest/README.md

```diff
@@ -0,0 +1,10 @@
+# Simple Dapp Testing
+In this example we have a Solidity contract, called `SimpleDapp`, allows users to deposit and withdraw Ether (ETH). This contract is essentially a simple ETH vault where users can safely deposit and later withdraw their funds. 
+
```

Comment:
I originally took the example from: https://github.com/Cyfrin/fuzz-testing-on-foundry 
We should reference it. 

Please also be aware that I had to remove a revert case from the original run (this line: https://github.com/Cyfrin/fuzz-testing-on-foundry/blob/main/test/SimpleDappTest.t.sol#L51) to make it pass. The failing run was here: https://prover.certora.com/output/53900/d068fc2a945b43c5b0defce31af63968/?anonymousKey=cb8a8f77e0206d0589549418923ac23632fea056 .

The call `simpleDapp.deposit{value: depositAmount}();` reverts - could you check if we could write the test such that it passes without changing the code?

Reply:
We have tried to execute it without changing the code, but failed...
It seems like it require many assumptions which we decided to omit.